Sources/Core/iOS/NetworkObserver.swift

```diff
@@ -82,19 +82,19 @@ private class NetworkIndicatorController {
     // Public API
 
     func networkActivityDidStart() {
-        assert(Thread.isMainThread(), "Altering network activity indicator state can only be done on the main thread.")
+        assert(Thread.isMainThread, "Altering network activity indicator state can only be done on the main thread.")
```

Comment:
Should we actually bump this to beta 3 now that it's out?
Because there are some additional changes to be done.